packages/design-system/src/components/tree/tree.tsx

```diff
@@ -333,50 +335,45 @@ const useKeyboardNavigation = <Data extends { id: string }>({
     return result;
   }, [root, getIsExpanded, getItemChildren]);
 
-  const rootRef = useHotkeys(
-    "up,down,right,left,space,backspace,delete,esc",
-    (event, { shortcut }) => {
-      if (selectedItem === undefined) {
-        return;
-      }
-      if (shortcut === "right" && getIsExpanded(selectedItem) === false) {
-        setIsExpanded(selectedItem, true);
-      }
-      if (shortcut === "left" && getIsExpanded(selectedItem)) {
-        setIsExpanded(selectedItem, false);
-      }
-      if (shortcut === "space") {
-        setIsExpanded(selectedItem, !getIsExpanded(selectedItem));
-      }
-      if (shortcut === "up") {
-        const index = flatCurrentlyExpandedTree.indexOf(selectedItem.id);
-        if (index > 0) {
-          setFocus(flatCurrentlyExpandedTree[index - 1], "changing");
-          event.preventDefault(); // prevent scrolling
-        }
-      }
-      if (shortcut === "down") {
-        const index = flatCurrentlyExpandedTree.indexOf(selectedItem.id);
-        if (index < flatCurrentlyExpandedTree.length - 1) {
-          setFocus(flatCurrentlyExpandedTree[index + 1], "changing");
-          event.preventDefault(); // prevent scrolling
-        }
-      }
-      if (shortcut === "backspace" || shortcut === "delete") {
-        onDelete(selectedItem.id);
+  const rootRef = useRef<HTMLDivElement | undefined>();
+
+  const handleKeyDown = (event: ReactKeyboardEvent) => {
+    // skip if nothing is selected in the tree
+    if (selectedItem === undefined) {
+      return;
+    }
+
+    // prevent scrolling
+    event.preventDefault();
```

Comment:
probably not every key cause scrolling? Why to prevent it

Reply:
useHotkeys added both preventDefault and stopPropagation but maybe you are right